DS4Windows/DS4Control/ControlService.cs

```diff
@@ -16,25 +16,22 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+using DS4Windows.DS4Control;
+using DS4WinWPF.DS4Control;
+using Microsoft.Win32;
+using Nefarius.ViGEm.Client;
+using Sensorit.Base;
+using SharpOSC;
 using System;
 using System.Collections.Generic;
-using System.IO;
-using System.Threading.Tasks;
-using System.Threading;
 using System.Diagnostics;
-using System.Windows.Threading;
+using System.IO;
```

Comment:
Code Cleanup is setup for every save for me so that's while there are large cosmetic changes..
I can redo these changes if you want.